services/spar/test-integration/Util/Scim.hs

```diff
@@ -617,3 +618,10 @@ urefFromBrig brigUser = case userIdentity brigUser of
           <> ", error = "
           <> e
   _ -> Nothing
+
+-- | The spar scim implementation makes use of its right to drop a lot of attributes on the
+-- floor.  This function calls the spar functions that do that.  This allows us to express
+-- what we expect a user that comes back from spar to look like in terms of what it looked
+-- like when we sent it there.
+whatSparReturnsFor :: HasCallStack => IdP -> Int -> Scim.User.User SparTag -> Either String (Scim.User.User SparTag)
+whatSparReturnsFor idp richInfoSizeLimit = either (Left . show) (Right . synthesizeScimUser) . validateScimUser' idp richInfoSizeLimit
```

Comment:
this might be a bit pedantic;  and question of philosophy on testing.

Is the property we want to test here   "spar distributes synthesizeUser over responses"  or  "Spar keeps these specific fields only" ? 

Reason I ask is that my expected workflow would be to change the test to expect the new field; get a failure, then implement the new feature. However with the way we formulate the test now; that isn't possible as the test will magically pass.

That's ok; but just wondering whether coupling what we're testing with the implementation is what we want here? It probably is?

Reply:
this is a very valid point.  i was wondering the same thing, wanna try duplicating the work in spar and see how ugly it gets?